### DISCUSSION TITLE
Open issue on EddieBot Repo: codeblocks for Usage field in ErrorEmbed

### DISCUSSION THREAD

**Discussion by Vyvy-vi** [EddieHubCommunity/support#547] [First-Timers] [answered]
##### **NOTE**- for First timers
It would be great if someone could open an issue on the [Eddiebot](https://github.com/EddieJaoudeCommunity/EddieBot/) repository or even open a PR for resolving the issue(after its issue is opened 🙂 ).  🟩 
Where to open issue: https://github.com/EddieJaoudeCommunity/EddieBot/issues

### Description of issue:
**Catagory**: Feature request/bug

**Brief Description**: Eddiebot's Command Error message embeds, suggest a `Usage` case of the particular command telling the user about how to use the command when things like bad arguments or missing arguments are passed. This Usage field is plain text, and it would be of much more utility if it was emphasized via  `code-blocks`.

### Additional Context: 
 - Screenshot:

The current usage text in command error embeds-
![Screenshot 2020-12-12 at 12 07 11 AM](https://user-images.githubusercontent.com/62864373/101941453-04a40980-3c0e-11eb-8344-774ed200d57b.png)

Better usage text format-
![Screenshot 2020-12-12 at 12 05 35 AM](https://user-images.githubusercontent.com/62864373/101941303-cb6b9980-3c0d-11eb-8487-2f6999eaebd9.png)

- **How to reproduce**: type any command with wrong arguments(like- `^standup .`)

- **Information regarding resolution of this**: (this might be useful, if provided in the opened issue)
   There are two ways to this(there may be more ways, but these are from the top of my head):
   1) the longer way: Edit the `usage` string in each file in `src/commandHandlers` folder, to format the usage string with code-blocks, by adding backticks around the commands text.
Example:
the current usage strings are like this->
![Screenshot 2020-12-11 at 11 43 13 PM](https://user-images.githubusercontent.com/62864373/101939199-a9bce300-3c0a-11eb-9b43-8fcd9521c91e.png)
this could be altered, to format with code-blocks on discord by slight modifications->
![Screenshot 2020-12-11 at 11 50 11 PM](https://user-images.githubusercontent.com/62864373/101939887-a5dd9080-3c0b-11eb-865e-78a3841216b6.png)

   2) Slightly quicker way: Run a format function on the usage variable, before adding that to the BuildErrorEmbed function:
Here, the `usage` string gets split into a list of different ways of using the command, by the `usageTextOptions` variable, then the `formatted` variable calls the `mapToCodeBlock` arrow function, to format the obtained list of usage into code-block formatted strings(with backticks), and then these usage cases get joined with an ` or ` in b/w and stored in `formatted` variable, which holds the string with the code-block formatting, ready to be passed into the `buildErrorEmbed` function:
![Screenshot 2020-12-11 at 11 57 29 PM](https://user-images.githubusercontent.com/62864373/101940581-b93d2b80-3c0c-11eb-9a1c-287d9d331fd8.png)

🤓  Thank you for contributing to EddieJaoudeCommunity!

**Comment by ChinmayMhatre** [comment]
I would like to do it !

**Reply by Vyvy-vi** [reply]
Oh.. @ChinmayMhatre I think @jveer634 opened an issue for this, but they opened on the wrong repo. [support#548](https://github.com/EddieJaoudeCommunity/support/issues/548)
I've requested them to open the issue on the EddieBot repo.

So, perhaps you could open the PR for this once they open an issue for this? :)

**Reply by ChinmayMhatre** [reply]
okay Thanks @Vyvy-vi  . I'll leave on comment on that issue as well  and create a pr  :)

**Reply by jveer634** [reply]
Sorry I didn't see your msg.. I opened an issue..

**Reply by ChinmayMhatre** [reply]
Is it on the eddie bot repo?

**Reply by Vyvy-vi** [reply]
yes it's there :)

**Comment by Vyvy-vi** [comment, marked as answer]
This issue was opened in [EddieBot#378](https://github.com/EddieJaoudeCommunity/EddieBot/issues/378)
by @jveer634 
🎉  :)

**Reply by Vyvy-vi** [reply]
The PR was opened in [Eddiebot#395](https://github.com/EddieJaoudeCommunity/EddieBot/pull/395)
by @ChinmayMhatre 
🎉 :)

**Reply by eddiejaoude** [reply]
Thanks 👍
